h/static/scripts/filter/test/document-title-test.js

```diff
@@ -1,75 +0,0 @@
-'use strict';
-
-var documentTitle = require('../document-title');
-
-describe('documentTitle', function() {
-
-  it('returns the title linked if the document has title and uri', function() {
```

Comment:
I think all the tests in here except the two "escapes HTML" tests will need equivalents for the new function - hopefully they should be somewhat simpler.